README.md

```diff
@@ -38,23 +38,47 @@ $ curl https://glide.sh/get | sh
 On Mac OS X you can also install the latest release via Homebrew:
 
 ```
-$ brew install glide
+$ brew install glide node
 ```
 
 On Ubuntu Precise (12.04), Trusty (14.04), Wily (15.10) or Xenial (16.04) you can install from our PPA:
 
 ```
-sudo add-apt-repository ppa:masterminds/glide && sudo apt-get update
-sudo apt-get install glide
+$ sudo add-apt-repository ppa:masterminds/glide && sudo apt-get update
+$ sudo apt-get install glide nodejs npm
 ```
 
-After that, you can build `updatehub-ce-server` as:
+After that, need to install `packr` that is a simple solution for bundling static assets inside of Go binaries use
+by `updatehub-ce-server`.
+
+To install Packr utility
+
+```
+$ go get -u github.com/gobuffalo/packr/packr
+```
+
+To get the dependency
+
+```
+$ go get -u github.com/gobuffalo/packr
+```
```

Comment:
Is this need? I think the first does it as well.

Reply:
Yes, need to install the booth